backend/app/migration/versions/6d5b798dd226_add_missing_industry_enum_values.py

```
@@ -0,0 +1,48 @@
+"""add missing industry enum values
+
+Revision ID: 6d5b798dd226
+Revises: d6bfa3df105a
+Create Date: 2025-06-30 17:30:45.370138
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
```

Ruff (0.11.9)

`sqlalchemy` imported but unused

Remove unused import: `sqlalchemy`

(F401)